lib/watch-sidebar.js

```diff
@@ -157,9 +170,10 @@ export default class WatchSidebar {
     document.removeEventListener("mouseup", this.resizeStopped);
   }
 
-  resizeSidebar({ pageX, which }) {
+  resizeSidebar({ pageX }: { pageX: number }, { which }: { which: number }) {
```

Comment:
This one blew my mind a little due to the destructuring. Appreciate if someone spot checks me here.